spec/javascripts/NewService/NewServiceForm.spec.jsx

```diff
@@ -6,34 +6,51 @@ import {NewServiceForm} from 'NewService'
 
 Enzyme.configure({adapter: new Adapter()})
 
-describe('New Service Form', () => {
-  const props = {
-    isServiceDiscoveryUsable: true,
-    serviceDiscoveryAuthenticateUrl: 'my-url',
-    providerAdminServiceDiscoveryServicesPath: 'my-path',
-    adminServicesPath: 'my-other-path'
-  }
-
-  it('should render properly', () => {
-    const wrapper = mount(<NewServiceForm {...props}/>)
-    expect(wrapper.find('#new_service_source').exists()).toEqual(true)
-    expect(wrapper.find(`input[name='source']`).length).toEqual(2)
-  })
-
-  it('should render new Service Manual form by default', () => {
-    const wrapper = mount(<NewServiceForm {...props}/>)
-    expect(wrapper.find('#new_service').exists()).toEqual(true)
-    expect(wrapper.find('#service_discovery').exists()).toEqual(false)
-  })
-
-  it('should render `Import from OpenShift` input not disabled when `isServiceDiscoveryUsable` is true', () => {
-    const wrapper = mount(<NewServiceForm {...props}/>)
-    expect(wrapper.find('#source_discover').props().disabled).toEqual(false)
-  })
-
-  it('should render `Import from OpenShift` input disabled when `isServiceDiscoveryUsable` is false', () => {
-    const notUsableProps = {...props, isServiceDiscoveryUsable: false}
-    const wrapper = mount(<NewServiceForm {...notUsableProps}/>)
-    expect(wrapper.find('#source_discover').props().disabled).toEqual(true)
-  })
+const props = {
+  isServiceDiscoveryUsable: true,
+  serviceDiscoveryAuthenticateUrl: 'authenticate-url',
+  providerAdminServiceDiscoveryServicesPath: 'my-path',
+  adminServicesPath: 'my-other-path'
+}
+const discoveryNotUsableProps = {
+  ...props,
+  isServiceDiscoveryUsable: false
+}
+
+it('should render itself', () => {
+  const wrapper = mount(<NewServiceForm {...props}/>)
+  expect(wrapper.find('#new_service_source').exists()).toEqual(true)
+  expect(wrapper.find(`input[name='source']`).length).toEqual(2)
+})
+
+it('should render new Service Manual form', () => {
+  const wrapper = mount(<NewServiceForm {...props}/>)
+  expect(wrapper.find('#new_service').exists()).toEqual(true)
+  expect(wrapper.find('#service_discovery').exists()).toEqual(false)
+})
+
+// TODO: remove `skip` when this is merged: https://github.com/airbnb/enzyme/pull/2008
+it.skip('should render new Service Discovery form when click on input', () => {
+  const wrapper = mount(<NewServiceForm {...props}/>)
+  wrapper.find('#source_discover').simulate('click')
+  wrapper.update()
+  expect(wrapper.find('#new_service').exists()).toEqual(false)
+  expect(wrapper.find('#service_discovery').exists()).toEqual(true)
+})
+
+it('should render `Import from OpenShift` input not disabled when `isServiceDiscoveryUsable` is true', () => {
```

Comment:
There are two things here:
1. The description of the test should be more "human", since it's meant to communicate what you will assert. This one looks more like pseudo code to me. You could go with something like `it('should render "Import from OS" input enabled when service discovery is selected', () =>...` 
2. I'd expect to see the text "Import from OpenShift" and that property `isServiceDiscoveryUsable` asserted since you are mentioning them in the description, but they are not... These apparently are, as I understand, not important and end up being a bit misleading (In this case)

This applies to the rest of the tests below.